### PR TITLE
feat: #188 add tooltip tests

### PR DESCRIPTION
**Pull request by barrydilan**
- [x]  content rendered
- [x]  too big content (text) hidden (it seems like this functionality is not yet implemented in the project)

### CODE REVIEW

File: mw-webapp/src/component/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -36,7 +36,10 @@ export const Tooltip = (props: PropsWithChildren<TooltipProps>) => {
       <span className={styles.target}>
         {props.children}
       </span>
-      <div className={classes}>
+      <div
+        className={classes}
+        data-cy="tooltipContent"
```
Comment (tritonJS826) [inline mw-webapp/src/component/tooltip/Tooltip.tsx:41]: move data-cy to the variable and reuse it in the test

File: mw-webapp/src/component/tooltip/Tooltip.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {Tooltip} from "src/component/tooltip/Tooltip";
+
+describe("Tooltip component", () => {
+  it("Content rendered", () => {
+    const tooltipContent = "Test content";
```
Comment (tritonJS826) [inline mw-webapp/src/component/tooltip/Tooltip.cy.tsx:5]: such constant should be declared with UPPER_CASE

Comment (tritonJS826) [inline mw-webapp/src/component/tooltip/Tooltip.cy.tsx:5]: lets play with slightly longer text

File: mw-webapp/src/component/tooltip/Tooltip.tsx (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/tooltip/Tooltip.tsx:41]: need add data-cy prop to tooltip props with optional flag.

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/tooltip/Tooltip.tsx (hunk 1 above)
Comment (tritonJS826) [reply mw-webapp/src/component/tooltip/Tooltip.tsx:41]: Agree, and then create variable in the test itself

File: mw-webapp/src/component/tooltip/Tooltip.module.scss [hunk 3]
```diff
@@ -13,6 +13,10 @@
 
   &:hover + .tooltip {
     display: block;
+    overflow: hidden;
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/tooltip/Tooltip.module.scss:16]: why you use overflow hidden for tooltip?

Comment (barrydilan) [reply mw-webapp/src/component/tooltip/Tooltip.module.scss:16]: We need overflow hidden in order to properly truncate text when its too big

File: mw-webapp/src/component/tooltip/Tooltip.cy.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    />);
+    cy.get(`[data-cy="${TOOLTIP_CONTENT_ID}"]`).should("contain.text", TOOLTIP_CONTENT);
+  });
+
+  it("Long content gets truncatet", () => {
+    cy.mount(<Tooltip
+      content={TOOLTIP_CONTENT_LONG}
+      testId={TOOLTIP_CONTENT_ID}
+    />);
+    cy.get(`[data-cy="${TOOLTIP_CONTENT_ID}"]`).invoke("show").then((text) => {
```
Comment (dayanych) [inline mw-webapp/src/component/tooltip/Tooltip.cy.tsx:22]: We have `getDataCy`, which return data attribute for CYPRESS testing. It is better to use the function: `cy.get(getDataCy(TOOLTIP_CONTENT_ID))`

Comment (dayanych) [reply mw-webapp/src/component/tooltip/Tooltip.cy.tsx:22]: After that, you can resolve conversation.

**Review by dayanych — APPROVED** [review]

File: mw-webapp/src/component/tooltip/Tooltip.module.scss (hunk 3 above)
Comment (SergioAJS) [reply mw-webapp/src/component/tooltip/Tooltip.module.scss:21]: Then how we can see the whole text?

Comment (Howl404) [reply mw-webapp/src/component/tooltip/Tooltip.module.scss:21]: Yea, kinda strange, full text should be visible

**Comment by tritonJS826** [comment]
abandoned

**Pull request closed without merging**